dashboard/pages/primary_screening/stages/s5_summary.py

```diff
@@ -1,4 +1,47 @@
-from dash import html
+from dash import dash_table, dcc, html
+from dash.dash_table.Format import Format, Scheme, Trim
+
+PRECISION = 5
+_ACT_INH_DATATABLE = dash_table.DataTable(
+    id="echo-bmg-combined",
+    columns=[
+        dict(id="CMPD ID", name="ID"),
+        dict(id="Destination Plate Barcode", name="Plate Barcode"),
+        dict(id="Destination Well", name="Well"),
+        dict(
+            id="% ACTIVATION",
+            name=" % ACTIVATION",
+            type="numeric",
+            format=Format(precision=PRECISION, scheme=Scheme.fixed),
+        ),
+        dict(
+            id="% INHIBITION",
+            name=" % INHIBITION",
+            type="numeric",
+            format=Format(precision=PRECISION, scheme=Scheme.fixed),
+        ),
+        dict(
+            id="Z-SCORE",
+            name=" Z-SCORE",
+            type="numeric",
+            format=Format(precision=PRECISION, scheme=Scheme.fixed),
+        ),
+    ],
+    style_table={"overflowX": "auto"},
+    style_data={
+        "padding-left": "10px",
+        "padding-right": "10px",
+        "width": "70px",
+        "autosize": {"type": "fit", "resize": True},
+        "overflow": "hidden",
+    },
```

Comment:
suggestion: consider adding `style_cell` argument as:
```python
style_cell={
    "font-family": "sans-serif",
    "font-size": "12px",
}
```
to use nicer font (imo more readable - I use it in my stage)